modules/users/ishika.tf

```diff
@@ -0,0 +1,12 @@
+data "discord_member" "Ishika" {
+  server_id = data.discord_server.server.id
+  user_id   = 1029378498449645578
+}
+
+resource "discord_member_roles" "Ishika" {
+  server_id = data.discord_server.er.id
```

Comment:
Great work @ishikkka !!! Could you just fix ; server_id = data.discord_server.server.id.
You didn't wrote server in that.